bbs_node:
  ros__parameters:
      max_range: 15.0

      # transformation search range
      min_tx: -50.0
      max_tx: 50.0
      min_ty: -50.0
      max_ty: 50.0
      min_theta: -3.15
      max_theta: 3.15

      # global map config
      # slice height range (map coordinate)
      map_min_z: 0.1
      map_max_z: 0.5
      # 2D map params
      map_width: 512
      map_height: 1024
      map_resolution: 0.3
      max_points_per_cell: 4
      # precomp map pyramid
      map_pyramid_level: 6

      # scan slice height range (sensor coordinate)
      scan_min_z: -0.2
      scan_max_z: 0.2
